source/Nevermore.Contracts/Nevermore.Contracts.csproj

```diff
@@ -21,6 +20,7 @@
     <GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">True</GeneratePackageOnBuild>
     <IncludeSymbols>true</IncludeSymbols>
     <WarningsAsErrors />
+    <TargetFramework>netstandard1.2</TargetFramework>
```

Comment:
Should we also include `netstandard2.0`?

Reply:
Not explicitly - by targeting `netstandard1.2` we should be forwards compatible with all of the same frameworks that are compatible with `netstandard2.0`